helm_deploy/hmpps-interventions-service/reports/team_kpi_report.sql

```diff
@@ -1,5 +1,6 @@
 WITH counts AS (
-    SELECT count(r.id) FILTER ( WHERE r.sent_at IS NULL )                   AS count_of_draft_referrals,
+    SELECT count(r.id)                                                      AS count_of_started_referrals,
```

Comment:
`count_of_started_referrals` instead of "all referrals" because when introduce date filtering, we'll do a `started_at` between earliest, latest